docs/en/overview/Architecture.md

```diff
@@ -257,6 +257,11 @@ configuring the property
 in the client. This section describes the behaviors of different write types as
 well as the performance implications to the applications.
 
+If you are writing with write type `MUST_CACHE`, `CACHE_THROUGH` or `ASYNC_THROUGH`,
```

Comment:
regardless of write types, files/dirs in Alluxio space always strongly consistent as all these write operations will go through Alluxio master first and modify Alluxio file system before return the the client/applications. So for applications talking to Alluxio, they will always see the latest update as long as the write operation complete successfully.

However, for users or applications taking the state in UFS into account, it might be different across write types:
- `MUST_CACHE` writes no data to UFS, so Alluxio space is never consistent with UFS
- `CACHE_THROUGH` writes data synchronously to Alluxio and UFS before returning success to applications. If writing to UFS is also strong consistent (e.g., HDFS), Alluxio space will be always consistent with UFS; if writing to UFS is eventual consistent (e.g. S3), it is possible that the file is written successfully to Alluxio but shown up in UFS later.
- `ASYNC_THROUGH` writes data to Alluxio and return to applications, leaving Alluxio to propagate the data to UFS asynchronously. From users perspective, the file can be written successfully to Alluxio, but get persisted in UFS later.
- `THROUGH` writes data to UFS directly without caching the data in Alluxio, however, Alluxio knows the files and its status.  Thus the metadata is still consistent.

Reply:
I put your comment into the `data consistency` subsection.